frontend/src/index.css

```diff
@@ -1,3 +1,10 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+@layer base {
+  :root {
+    --color-dark-background: #10172a;
+    --color-dark-textfield-card: #1d293c;
```

Comment:
_Disclaimer: I haven't been following this conversation. So, apologies if things get repetitive_.

@krishnaacharyaa from where are we using this specific color? I don't see it anywhere in the official [tailwind theme colors](https://tailwindcss.com/docs/customizing-colors). The closest ones are either gray-900 or slate-900.

I suggest we stick to the official colors and change both of the colors to slate or gray schemes. Can you check it and confirm?

Reply:
@chinmaykunkikar  I used this color from the figma file provided by the @krishnaacharyaa  
"https://www.figma.com/file/zqNcWGGKBo5Q2TwwVgR6G5/WanderLust--A-Travel-Blog-App?type=design&node-id=148-2&mode=design&t=ylnaldxFn5jMgyWI-0"

You can check.
First I tryed to used pre define color but these colors have some difference

Reply:
@chinmaykunkikar  I used these specific color from the figma file provided by the @krishnaacharyaa 
 
https://www.figma.com/file/zqNcWGGKBo5Q2TwwVgR6G5/WanderLust--A-Travel-Blog-App?type=design&node-id=148-2&mode=design&t=ylnaldxFn5jMgyWI-0

you can check.

Reply:
@chinmaykunkikar  can u resolve my issue too.

Reply:
@SankalpHaritash21 what issue would that be?

Reply:
@chinmaykunkikar, actually @SankalpHaritash21 came up with closest tw colors (claimed) but it was too off,  So I gave him idea by asking him to refer dark theme referring to the tw official docs https://github.com/krishnaacharyaa/wanderlust/pull/43#issuecomment-1788456745 which looked cool to me, but i am still open to the official tw-colors as far it looks good.

Reply:
Ok @krishnaacharyaa  then I will directly use these color 
@chinmaykunkikar can u tell me what's the background color,  card color and text field color

Reply:
@krishnaacharyaa  which version should I use v1 Or v2

Reply:
@SankalpHaritash21 what version ? in the above context and he has clearly mentioned the colors with hex code and the tw-class names, kindly refer the same for the bg color , card color and text field color, This has solution to all of your problems https://github.com/krishnaacharyaa/wanderlust/pull/62#discussion_r1408422049